lib/chef/knife/ec_backup.rb

```diff
@@ -252,6 +262,6 @@ def chef_fs_copy_pattern(pattern_str, chef_fs_config)
              Chef::ChefFS::FileSystem::OperationFailedError => ex
         knife_ec_error_handler.add(ex)
       end
-    end
-  end
+end
+end
 end
```

Comment:
This doesn't appear to be correctly formatted with 3 `end` 's at the same level.